.travis.yml

```diff
@@ -3,6 +3,7 @@ language: php
 php:
   - 7.2
   - 7.4
+  - 8.1
```

Comment:
let's  create another PR were we are switching to github acttions. You can use this as example: https://github.com/mmoreram/GearmanBundle/blob/master/.github/workflows/test.yaml

Reply:
I will look into and create another PR. Thank you.